.circleci/config.yml

```diff
@@ -1,10 +1,38 @@
 version: 2
+
+workflows:
+  version: 2
+  workflow:
+    jobs:
+      - test-3.6
+
+setup:
+  - &lint
+    run:
+      name: Run flake8
+      command: |
+        black --check .
+        flake8
+        isort -rc -c .
+        
+defaults: &defaults
+  working_directory: ~/code
+  steps:
+  - checkout
+  - run:
+      name: Install black flake8 and isort
+      command: pip install flake8 flake8-docstrings flake8-polyfill pep8 pep8-naming isort black==19.10b0
+  - *lint
+  - run:
+      name: Install dependencies
+      command: pip install -r requirements.txt
+  
+  - run:
+      name: Test
+      command: PYTHONPATH=. python tests/__init__.py
+
 jobs:
-  build:
+  test-3.6:
+    <<: *defaults
     docker:
-      - image: circleci/python:3.5.7
```

Comment:
We need to run these tests in 3.5 and it would be nice to also run in 3.6. So you can run black as a separate job in the workflow using Python 3.6 just for that check.